content/docs/configuration/acme/dns01/route53.md

```diff
@@ -47,30 +47,382 @@ permissions:
 }
 ```
 
-> Note: The `route53:ListHostedZonesByName` statement can be removed if you
+> ℹ️ The `route53:ListHostedZonesByName` statement can be removed if you
 > specify the (optional) `hostedZoneID`. You can further tighten the policy by
 > limiting the hosted zone that cert-manager has access to (e.g.
 > `arn:aws:route53:::hostedzone/DIKER8JEXAMPLE`).
+>
+> 📖 Read about [actions supported by Amazon Route 53](https://docs.aws.amazon.com/Route53/latest/APIReference/API_Operations_Amazon_Route_53.html),
+> in the [Amazon Route 53 API Reference](https://docs.aws.amazon.com/Route53/latest/APIReference/Welcome.html).
+>
+> 📖 Learn how [`eksctl` can automatically create the cert-manager IAM policy](https://eksctl.io/usage/iam-policies/#cert-manager-policy), if you use EKS.
 
 ## Credentials
```

Comment:
In https://github.com/cert-manager/website/issues/753 @AlverezYari wrote:
> Route53 - AWS IAM Account Setup is confusing

I've tried to summarize all the options in this revision of the documentation.

@AlverezYari Please give feedback if you have time:
 * https://deploy-preview-1555--cert-manager.netlify.app/docs/configuration/acme/dns01/route53/#credentials